src/components/KonnectorSuccess.spec.jsx

```diff
@@ -1,18 +1,39 @@
 import React from 'react'
-import { mount } from 'enzyme'
-import KonnectorSuccess, { BanksLink, DriveLink } from './KonnectorSuccess'
+import { render, screen } from '@testing-library/react'
+
 import AppLike from 'test/AppLike'
+import KonnectorSuccess from './KonnectorSuccess'
+
+// @TODO: some pretty ugly mocks here because the app uses different react-redux versions
+jest.mock('react-redux/lib/utils/Subscription', () => ({
+  createSubscription: () => ({
+    trySubscribe: () => jest.fn(),
+    tryUnsubscribe: () => jest.fn(),
+    notifyNestedSubs: () => jest.fn()
+  })
+}))
+
+jest.mock(
+  'cozy-client/node_modules/react-redux/lib/utils/Subscription',
+  () => ({
+    __esModule: true,
+    default: () => ({
+      trySubscribe: () => jest.fn(),
+      tryUnsubscribe: () => jest.fn()
+    })
+  })
+)
 
 describe('KonnectorSuccess', () => {
-  let trigger, connector, root
+  let trigger, connector
   const fakeStore = {
     banksUrl: 'https://example-banks.mycozy.cloud',
     getState: () => ({}),
     subscribe: () => ({}),
     dispatch: () => ({})
   }
   const setup = () => {
-    root = mount(
+    render(
```

Comment:
nit: shouldn't we `return` the render and deconstruct it instead of using `screen` on tests?

Reply:
@Ldoppea I went "back to basics" and used the RTL documentation. In it, many examples use `screen` for simple tests unless something specific is needed. What do you think? also cc @trollepierre

Reply:
see also https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
if it's recommended by the author of the library, let's trust him.